CHANGELOG.rst

```diff
@@ -5,6 +5,44 @@ Synapse Changelog
 *****************
 
 
+v2.83.0 - 2022-02-17
+====================
+
+Features and Enhancements
+-------------------------
+- Add ``:ip:proto`` and ``:ip:tcp:flags`` properties to the ``inet:flow``
+  form.
+  (`#2554 <https://github.com/vertexproject/synapse/pull/2554>`_)
+- Add ``$lib.log.debug()``, ``$lib.log.info()``, ``$lib.log.warning()``, and
+  ``$lib.log.error()`` Stormtypes APIs. These allow a user to send log
+  message to the Cortex output directly.
+- Add support for the ``synapse.tools.genpkg`` tool to support using files
```

Comment:
The "add support to support" reads a little weird to me, maybe
"Update the ``synapse.tools.genpkg`` tool to support using files with the ``.storm`` extension"
or
"Add support for the ``synapse.tools.genpkg`` tool to use files with the ``.storm`` extension"

Reply:
fixed